Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -158,6 +159,7 @@ @interface ConversationViewController () <AttachmentApprovalViewControllerDelega
 @property (nonatomic) TSThread *thread;
 @property (nonatomic) YapDatabaseConnection *editingDatabaseConnection;
```

Comment:
Don't we want to get rid of the `editingDatabaseConnection` property? For that matter, shouldn't we get rid of `uiDatabaseConnection` too?

Reply:
Good catch, I'll make them readonly / computed.